docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -289,13 +289,21 @@ Physical nodes (or cloud instances) are installed with a standard CentOS 7 serve
 1. Add a new `yugabyte:yugabyte` user and group with the default login shell `/bin/bash` that you set via the `-s` flag, as follows:
 
     ```bash
-    sudo useradd -s /bin/bash -m yugabyte   # (add user yugabyte and create /home/yugabyte)
+    sudo useradd -s /bin/bash --create-home --home-dir <yugabyte-home> yugabyte  # (add user yugabyte and create /home/yugabyte)
```

Comment:
Change `# (add user yugabyte and create /home/yugabyte)` to `# (add user yugabyte and create its home directory as specified in <yugabyte-home>)`